.github/workflows/toggle-egress.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: toggle egress proxy
+
+on:
+  workflow_dispatch:
+    inputs:
+      toggle-egress:
+        description: 'Enable/disable egress proxy'
+        required: true
+        type: choice
+        options:
+          - "enable"
+          - "disable"
+      appName:
+        description: 'App Name'
+        required: true
+        type: choice
+        options:
+          - "catalog-web"
+          - dashboard
+          - "dashboard-stage"
+    environ:
+      description: 'Cloud Foundry Space'
+      required: true
+      type: choice
+      options:
+        - development
+        - staging
+        - prod
+
+jobs:
+  toggle-egress:
+    concurrency: ${{ github.event.inputs.environ }}
+    name: ${{github.envent.inputs.toggle-egress}} egress (${{ github.event.inputs.environ }})
+    environment: ${{ github.event.inputs.environ }}
```

Comment:
Did Tyler or Jin already change the Github Action Environments?

Reply:
I did not change the 'production' to 'prod' in data.gov